soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -317,6 +318,7 @@ __weak void pm_state_exit_post_ops(enum pm_state state, uint8_t substate_id)
 		z_xtensa_cache_flush_inv_all();
 		z_xt_ints_on(core_desc[cpu].intenable);
 
+		mem_window_idle_exit();
```

Comment:
Definitely only when IMR context saving is enabled and by the primary core only.